### ISSUE TITLE
Conditional Variable groupColor might have wrong documentation or code for its permissions

### ISSUE THREAD

**Issue by noiverre** [CryptoMorin/KingdomsX#51]
**How to reproduce**

1. give player X OP perms
2. chat, the color will return perm_color_admin color which is &6 by default
4. deop player X
5. define new groupColor, perm_color_test with color &9
6. create new group in LuckPerms named testGroup
7. give permission color.test to group testGroup
8. assign player X to testGroup
9. make player X chat something
10. it returns else value (&7 by default) instead of perm_color_test value

![image](https://user-images.githubusercontent.com/77881970/226907658-c0411a5f-e247-4183-b2fe-7876cca4dcf1.png)
![image](https://user-images.githubusercontent.com/77881970/226907733-15d0a9ca-6228-454b-9e0f-d67bc0be2290.png)

**Comment by noiverre** [comment]
chat.yml
```yaml
channels:
  # WARNING: DO NOT change the IDs of default channels.
  global: # this is the channel's ID
    # You can also use LuckPerms contextual permissions to disable these in certain worlds.
    # Conditions the player must have in order to use this channel.
    color: '&f'

    # This is used for console and DiscordSRV.
    # The colors are obviously ignored for DiscordSRV.
    # This option is also used for that chat event's formatting (only used when cancel option is set to false)
    admin-formats:
      kingdoms_has_kingdom: &standard-admin-format '{$channel}&7|{$nationPrefix}{$kingdomPrefix}%displayname% &8≫ {$groupColor}%message%'
      else: '{$channel} %displayname% &8≫ {$groupColor}%message%'

    # This is the formats used for the chat. The entries are the conditions.
    # To make conditions less confusing you can also use "placeholder formats" check them on the Wiki.
    formats:
      kingdoms_has_kingdom: "{$nationPrefix}{$kingdomPrefix}%displayname% &8≫ {$groupColor}%message%"
      else: "hover:{%displayname%;&2Message %player%;|/msg %player% } &8≫ {$groupColor}%message%"
  kingdom:
    # If the condition is not met, the message path specified will be shown to the player.
    # If this option is not specified, all players will be able to use them.
    use-conditions:
      kingdoms_has_kingdom: 'no-kingdom.default'

    # Conditions that the player must have in order to receive the messages
    # If this option is not specified, all players will be able to see the messages.
    recipients-condition: "kingdoms_is_spy || kingdoms_relation == 'SELF'"
    color: '&2'

    admin-formats: *standard-admin-format
    formats: "{$spy}{$nationPrefix}{$kingdomPrefix}%displayname% &7≫ &d%message%"
``

config.yml
```yaml
    ### Conditional Variables ###
    groupColor:
      #perm_color_admin: '&6'
      perm_color_professor: '&#FFFFFF'
      perm_color_senior: '&#ECECEC'
      perm_color_regular: '&#D2D2D2'
      perm_color_junior: '&#B4B4B4'
      perm_color_newbie: '&#989898'
      perm_color_guest: '&#7B7B7B'
      else: '&7'
```

**Comment by CryptoMorin** [comment]
Fixed in v1.16.3
